blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+  powering off. The SCI interrupts have been disabled on all PCI Express Root
+  Ports, until proper handling of SCI is implemented to fix the problem.
+
+- GPIO controller ACPI device yellow bang in Windows device manager
+
+  Despite previous attempts to fix the yellow bang of GPIO controller in
+  Windows Device Manager, the issue still persisted. The ACPI GPIO controller
+  device had all the definitions of GPIOs for Alder Lake-S, but apparently,
+  Windows was unable to properly initialize the GPIO because of the silent
+  dependency on the ACPI path of the device in the Windows GPIO driver. While
+  Linux had no problems with previous ACPI path, Windows could not properly
+  load drivers in correct sequence, thus the ACPI path had to be changed from
+  `\_SB.PCI0.GPIO` to `\_SB.GPIO` so that Windows will not depend on
+  enumeration and initialization of `\_SB.PCI0` ACPI device. Similar issues
+  were observed with TPM2 device ACPI path. When set wrong, the Windows 11
+  installer would not work. Now the yellow bang in Device Manager should be
+  gone for good.
+
+- Resource conflicts with chipset internal P2SB PCI device being incorrectly
+  defined and initialized in coreboot
+
+  Issue observed during the 2x dGPU memory allocation problems and GPIO
+  allocation problems. It occurred that coreboto did not reserve proper memory
+  range for one of the chipset internal PCI devices resulting in resource
+  conflicts and issues with overall operation of conflicting devices. The issue
+  could be detected earlier if not the FSP which hides this internal device
+  during silicon initialization phase. Hidden device could not be detected by
+  coreboot resource allocator and thus the reserved memory for this device was
+  not accounted for.
+
+- Reset button hanging the platform for up to 2 minutes due to watchdog bug
+
+  Noticed by Dasharo Team developer and also reported by Qubes OS community.
+  Not exactly a regression, but a bug in the OC watchdog hardware which did not
+  allow setting a small timeout to expire the watchdog after an unexpected
+  reset (a reset button press for example). Initially the OC watchdog was
+  designed to detect unstable overclocking configurations, when such board
+  unexpectedly reset, the BIOS was supposed to detect it and act per policy to
+  restore safe configuration. Dasharo intended to use it in the same way but
+  the timeout not updating bug in the watchdog hardware caused the firmware to
+  wait whole 2 minutes for watchdog expiration after reset button press. Now
+  Dasharo firmware does not detect unexpected resets and simply initializes the
+  watchdog by programming the timeout. Expiration is not handled in any way.
+
+# Current state of Qubes OS R4.1.2 on MSI PRO Z690-A with Dasharo firmware
+
+Here is a short demo of Qubes OS R4.1.2-rc1 running on the newest Dasharo
+v1.1.1 compatible with MSI PRO Z690-A.
+
+TBD: YT link?
```

Comment:
What I'm missing in this whole message? How I can get this shiny wonderful firmware release? How I can safely install it on my machine that I bought with Dasharo Supporters Entrance subscription.

Reply:
DTS does not yet allow to update to v1.1.1 right? @TomaszAIR 
So this comment would block the post release...

Reply:
Added section how to update: https://github.com/3mdeb/news-and-ideas/pull/513/commits/0f29e7a980e1814bfdf6c3d8ab045c7ba6d63782#diff-b67e40bb2c6780be37b7e6cf6fc2a2a9f9ad977388795cc0c5bdfa6576dc52b6R193